packages/headless/src/app/commerce-engine/commerce-engine.ts

```diff
@@ -22,7 +22,11 @@ import {cartReducer} from '../../features/commerce/context/cart/cart-slice';
 
 export type {CommerceEngineConfiguration};
 
-const commerceEngineReducers = {productListing: productListingV2Reducer, commerceContext: contextReducer, cart: cartReducer};
+const commerceEngineReducers = {
```

Comment:
looks like bunch of commerce files (which probably were not lint staged) got included guessing due to lint staging of master merge commit